### PR TITLE
Add traces to LowerGradOf and SpecializeAutoGrad

### PR DESCRIPTION
**Pull request by Krovatkin**

### CODE REVIEW

File: torch/csrc/jit/jit_log.cpp [hunk 1]
```diff
@@ -14,6 +15,10 @@ JitLoggingLevels jit_log_level() {
   return log_level;
 }
 
+std::string debugValueOrDefault(Node* n) {
```
Comment (ZolotukhinM) [inline torch/csrc/jit/jit_log.cpp]: Could we pass `const Node&` here?

**Review by ZolotukhinM — APPROVED** [review]
Looks good!

File: torch/csrc/jit/jit_log.cpp (hunk 1 above)
Comment (Krovatkin) [reply torch/csrc/jit/jit_log.cpp]: lettuce try.

**Comment by ezyang** [comment]
This commit broke Windows https://ci.pytorch.org/jenkins/job/caffe2-builds/job/py2-cuda9.0-cudnn7-windows-build/54870//console

**Pull request closed without merging**